src/config/babelrc.js

```diff
@@ -65,7 +65,6 @@ module.exports = function (api, opts = {}) {
         require('@babel/plugin-transform-runtime').default,
         {
           helpers: false,
-          polyfill: false,
```

Comment:
I don't know how it relates to this change, I just want to make sure this removal was intentional.

Reply:
is intentional that option is no longer available